### ISSUE TITLE
ERROR: Error installing climate_control: activesupport requires Ruby version >= 2.2.2

### ISSUE THREAD

**Issue by sapslaj** [thoughtbot/climate_control#19]
When trying to install climate_control on a Ruby version older than 2.2.2, ActiveSupport refuses to install since Rails 5, and hence ActiveSupport 5, requires 2.2.2 or newer. 

```
[sapslaj@rando ~ ]$ docker run -it ruby:2.1.9 bash
root@3f2a8972eac9:/# ruby -v
ruby 2.1.9p490 (2016-03-30 revision 54437) [x86_64-linux]
root@3f2a8972eac9:/# gem install climate_control  
Fetching: i18n-0.7.0.gem (100%)
Successfully installed i18n-0.7.0
Fetching: thread_safe-0.3.5.gem (100%)
Successfully installed thread_safe-0.3.5
Fetching: tzinfo-1.2.2.gem (100%)
Successfully installed tzinfo-1.2.2
Fetching: minitest-5.9.0.gem (100%)
Successfully installed minitest-5.9.0
Fetching: concurrent-ruby-1.0.2.gem (100%)
Successfully installed concurrent-ruby-1.0.2
Fetching: activesupport-5.0.0.gem (100%)
ERROR:  Error installing climate_control:
    activesupport requires Ruby version >= 2.2.2.
root@3f2a8972eac9:/# 

```

**Comment by dentarg** [comment]
This should be solved with the recently released 0.1.0 version :)

```
$ ruby -v
ruby 2.1.10p492 (2016-04-01 revision 54464) [x86_64-darwin15.0]

$ gem install climate_control
Fetching: climate_control-0.1.0.gem (100%)
Successfully installed climate_control-0.1.0
Parsing documentation for climate_control-0.1.0
Installing ri documentation for climate_control-0.1.0
Done installing documentation for climate_control after 0 seconds
1 gem installed
```

**Comment by geoffharcourt** [comment]
Closing, as this appears solved.
